doc/development/plugins.rst

```diff
@@ -183,7 +183,12 @@ To execute operations on the device, the following methods **must** be defined:
 .. autosummary::
 
     apply
-    probability
+
+If the device is analytic then it **must** also overwrite:
```

Comment:
Maybe useful to explain "device is analytic", for example referring to the flag ``analytic = True''? Only if it makes sense...